modules/apps/adaptive-media/adaptive-media-document-library-thumbnails/src/main/java/com/liferay/adaptive/media/document/library/thumbnails/internal/processor/AMImageEntryProcessor.java

```diff
@@ -64,34 +59,10 @@
 @Component(
 	configurationPid = "com.liferay.adaptive.media.document.library.thumbnails.internal.configuration.AMSystemImagesConfiguration",
 	property = "type=" + DLProcessorConstants.IMAGE_PROCESSOR,
-	service = {
-		AMImageEntryProcessor.class, DLProcessor.class, ImageProcessor.class
-	}
+	service = ImageProcessor.class
 )
-public class AMImageEntryProcessor implements DLProcessor, ImageProcessor {
-
-	@Override
-	public void afterPropertiesSet() {
-	}
-
-	@Override
-	public void cleanUp(FileEntry fileEntry) {
-	}
-
-	@Override
-	public void cleanUp(FileVersion fileVersion) {
-	}
-
-	@Override
-	public void copy(
-		FileVersion sourceFileVersion, FileVersion destinationFileVersion) {
-	}
-
-	@Override
-	public void exportGeneratedFiles(
-		PortletDataContext portletDataContext, FileEntry fileEntry,
-		Element fileEntryElement) {
-	}
+public class AMImageEntryProcessor
+	extends AMDLProcessor implements ImageProcessor {
```

Comment:
Hey @RenanUmV, do not extend the `AMDLProcessor` here... just copy the fields amImageMimeTypeProvider and  amImageValidator here.

Thx

Reply:
Hey @RenanUmV, never mind. As we discussed today, it will require implementing some methods so It's not worth it in my point of view.

Thanks for catch that